src/Controller/Admin/TournamentController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Participant;
+use App\Entity\Player;
+use App\Entity\Tournament;
+use App\Form\Type\AddParticipantType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class TournamentController extends Controller
+{
+    public function addPlayersTournamentAction(Request $request, $id)
+    {
+        $tournament = $this->getDoctrine()->getRepository(Tournament::class)->find($id);
+        $participants = $this->getDoctrine()->getRepository(Participant::class)->findBy(['tournament' => $id]);
+
+        $players = $this->getDoctrine()->getRepository(Player::class)->findAll();
+        $formOptions = [];
+        foreach ($players as $player) {
+            $ready = false;
+            foreach ($participants as $participant) {
```

Comment:
It's not very efficient approach. Try to move the selection to the database. Use "where ... NOT IN ... " sql condition through doctrine. Please refer to the examples 
https://www.ozonesolutions.com/programming/2011/09/symfony-doctrine-where-in-and-where-not-in-syntax/